tests/e2e/run-e2e-tests.sh

```diff
@@ -129,6 +129,9 @@ if [[ ! -f "/i-am-rox-ci-image" ]]; then
     kubeconfig="${KUBECONFIG:-${HOME}/.kube/config}"
     mkdir -p "$QA_TEST_DEBUG_LOGS"
     info "Running in a container..."
+    # .gitconfig needs to mounted from a location that is shared with the
+    # linux VM running on macOS. Hence we cannot use /tmp/gitconfig here.
+    printf '[safe]\n\tdirectory = "%s"' "${ROOT}" > "$ROOT/.gitconfig"
```

Comment:
Um, do we really want to keep adding entries on every invocation? And without trailing newline?

Reply:
`>` overwrites the file`, `>>` appends.

Reply:
One could add if not exist (untested):

```shell
grep -q '[safe]\n\tdirectory = "%s"'  "$ROOT/.gitconfig" || printf '[safe]\n\tdirectory = "%s"' "${ROOT}" >> "$ROOT/.gitconfig"
```

Reply:
No `.gitconfig` is just a name here. It is not the `.gitconfig` of the host system that lives in `$HOME/.gitconfig` or `$HOME/.config/git/config`, but just a file that is written to the project root and then mounted into the container. The reason I put the file into the project root is because it is guaranteed to be shared with the linux VM - as opposed to for example `/tmp/.gitconfig`.

Reply:
Also the problem with checking the file into git is that `[safe]...` has the full path of the repo. So it's different for everyone, in my case it's
```
[safe]
	directory = "/Users/stephan/go/src/github.com/stackrox/stackrox"

```

Reply:
Oh, right!